source/includes/steps-install-shell-windows.yaml

```diff
@@ -0,0 +1,60 @@
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: navigate-dlc
+ref: navigate-dlc-windows
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: open-mdb-shell-page
+ref: open-mdb-shell-page-windows
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: download-archive
+ref: download-archive-windows
+---
+title: "Extract the files from the downloaded archive."
+ref: extract-archive
+level: 4
+---
+title: "Add the MongoDB Shell binary to your ``PATH`` environment
+   variable."
+ref: add-shell-to-path
+level: 4
+content: |
+
+   Ensure that the extracted |mdb-shell| binary is in the desired 
+   location in your filesystem, then add that location to your ``PATH`` 
+   environment variable.
+
+   To add the |mdb-shell| binary's location to your 
+   ``PATH`` environment variable:
+
+   1. Open the :guilabel:`Control Panel`.
+
+   #. In the :guilabel:`System and Security` category, click 
+      :guilabel:`System`.
+
+   #. Click :guilabel:`Advanced system settings`. The **System 
+      Properties** modal displays.
+
+   #. Click :guilabel:`Environment Variables`.
+
+   #. In the *System variables* section, select ``Path`` and click 
+      :guilabel:`Edit`. The **Edit environment variable** modal 
+      displays.
+
+   #. Click :guilabel:`New` and add the filepath to your ``mongosh`` 
+      binary.
+
+   #. Click :guilabel:`OK` to confirm your changes. On each other 
+      modal, click :guilabel:`OK` to confirm your changes.
+
+   To confirm that your ``PATH`` environment variable is correctly 
```

Comment:
Since you don't specify how to extract the archive in step 4, I'd also consider omitting how to open a command prompt, since there are multiple ways to do that, too. But perhaps I'm just used to doing it and it's not as simple as I think?

Reply:
Good call, I agree. Removed that part.